cms/static/coffee/spec/models/section_spec.js

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I noticed that once you do this, all that is left are the test fixtures. It would be good to move those too so that the ``coffee`` directories can be completely removed.